### DISCUSSION TITLE
New icon

### DISCUSSION THREAD

**Discussion by pmalicki11** [twbs/bootstrap#38972] [Ideas] [answered]
According to twitter rebranding to x this new icon will be needed.

**Comment by julien-deramond** [comment]
Could you please create an [Icon Request](https://github.com/twbs/icons/issues/new?assignees=&labels=icon-request&projects=&template=icon-request.md&title=) instead (if this one doesn't exist already), and then delete this discussion (or close it if not possible). It will be easier for our backlog to have it with the other requests :) Thx

**Comment by pmalicki11** [comment, marked as answer]
Sorry for that: Here is the issue: https://github.com/twbs/icons/issues/1778
